android/src/main/java/com/gaspardbruno/staticsafeareainsets/RNStaticSafeAreaInsetsModule.java

```diff
@@ -47,4 +55,39 @@ public String getName() {
 
     return constants;
   }
+
+  @ReactMethod
+  public void getSafeAreaInsets(Callback cb) {
+    // Approach A
```

Comment:
I tried *approach A* without success. Managed to get values with *approach B* but I'm not happy with the repetition.